lib/site_settings/type_supervisor.rb

```diff
@@ -35,7 +35,8 @@ def self.types
       group_list: 20,
       tag_list: 21,
       color: 22,
-      simple_list: 23
+      simple_list: 23,
+      html: 24
```

Comment:
you will have to bump this, I just merged emoji_list which is using this 24 position, sorry 😅